contracts/sfc/SFC.sol

```diff
@@ -392,6 +400,7 @@ contract SFC is Initializable, Ownable, Version {
         emit UpdatedSlashingRefundRatio(validatorID, refundRatio);
     }
 
+    // Recount votes for a delegator and a validator
     function recountVotes(address delegator, address validatorAuth, bool strict, uint256 gas) external {
         // solhint-disable-next-line avoid-low-level-calls
```

Comment:
Please add reasoning - we must prevent possibility of governnace contract to break epoch sealing.

Reply:
I will put this into another PR